Cloudinary/Operation/02_CloudinaryMapping.md

```diff
@@ -0,0 +1,135 @@
+
+> [Info] The following procedure describes exemplary how to map a Cloudinary image attribute. The procedure is largely identical to map a Cloudinary video attribute. The differences are explicitly specified at the appropriate place.
+
+1. Click the attribute set mapping you want to edit in the list of attribute set mappings.   
+  The *Mapping from source set to destination set* view is displayed.
+
+  ![Mapping](../../Assets/Screenshots/DataHub/Settings/ETL/Mapping.png "[Mapping]")
+
+2. Click the row with the destination attribute to which you want to map the Cloudinary image attribute. The destination attribute must have either the *String* data type or the *Cloudinary image* data type.  
+  The *Settings* section is displayed on the right side.
+
+  ![Cloudinary source attribute](../../Assets/Screenshots/DataHub/Settings/ETL/CloudinarySourceAttribute.png "[Cloudinary source attribute]")
+
+  > [Info] Depending on the selected row, the fields displayed in the *Settings* section differ:
+  - If the selected row contains only a destination attribute, the destination attribute as well as the *Extension* and *Override change tracking mode* drop-down lists are displayed.
+  - If the selected row contains an attribute mapping with source, destination attribute and the extension, the destination attribute as well as the *Source attribute*, *Extension* and *Override change tracking mode* drop-down lists are displayed. The *Source attribute*, *Extension* and *Override change tracking mode* drop-down lists are preset. If the selected extension requires further configuration settings, the *Configuration* section is displayed below the *Settings* section.  
+
+
+3. Select the appropriate option in the *Extension* drop-down list in the *Settings* section. The following extensions are available to map a Cloudinary source attribute with:
+  - **Copy cloudinary image value**: Copy the value from the Cloudinary image attribute and map it to another Cloudinary image attribute. This extension is only available for a destination attribute with the *Cloudinary image* data type.
+  - **Cloudinary to string**: Map a Cloudinary image to a string attribute. If desired, the image transformation can also be mapped to the destination attribute. This extension is only available for a destination attribute with the *String* data type.
+
+  The *Configuration* section is displayed below the *Settings* section.
+
+  > [Info] For detailed information about the ETL extensions, see [ETL extensions list](../../DataHub/UserInterface/03_ETLExtensions.md).   
+  For detailed information about the data types, see [Data type list](../../PIM/UserInterface/04_DataTypeList.md).
+
+
+4. Select the appropriate tracking mode in the *Override change tracking mode* drop-down list. By default, the **Use default** option is preselected. The following modes are available:
+  - **Use default**: ---
```

Comment:
Vielleicht Beschreibung wie " The default values are applied/used" o.ä. hinzufügen.